frontend/src/services/groupSettingsService.ts

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios';
+import { GroupsConfig } from '../types';
+
+export const fetchGroupsSettings = (): Promise<GroupsConfig> => {
+  const url = '/api/groups-config';
+  return axios
+    .get(url)
+    .then((response) => {
+      return response.data;
+    })
+    .catch((e) => {
+      throw new Error(e.response.data.message);
+    });
+};
+
+export const updateGroupsSettings = (
+  settings: GroupsConfig,
+): Promise<{ success: GroupsConfig | null; error: string | null }> => {
+  const url = '/api/groups-config';
+  return axios
+    .post(url, settings)
```

Comment:
I wonder if this would be more `put` as you're not ever "creating". REST is weird. I'm okay if you keep this as post. It will mean something different in the SDK when we get to it.

Reply:
mmm you're completly right, i never create it, it's definitely PUT.